snippets/index.js

```diff
@@ -0,0 +1,75 @@
+'use babel';
```

Comment:
I would like to discuss all of these snippets code. We don't use it at this moment.
But it is probably increasing the atom package size.

Do you know if you comment on this code, then will the whole Pymakr JS code be minified in production and commented code won't be included in the production release?

Reply:
I don't know. Because there's no need to compile the code. I'm not sure how it works. But Pymakr VSC it does need to be compiled and the compiled files aren't minified at all. So probably it's the same here on pymakr Atom.

Reply:
I am not sure if that would help a lot, but it might be a good idea to minify the files in the future. To have a smaller pymakr download size.

Reply:
Probably it won't save a lot, since pymakr itself is already very small. The problem is the SerialPort binaries. But I agree it would be better to implement a smarter project structure